src/Symfony/Component/DependencyInjection/Alias.php

```diff
@@ -125,6 +125,11 @@ public function getDeprecationMessage(string $id): string
         return str_replace('%alias_id%', $id, $this->deprecationTemplate ?: self::$defaultDeprecationTemplate);
     }
 
+    public function getDeprecationTemplate(): ?string
```

Comment:
The same concept exists on `Definition` but this method doesn't exists there, which means it should be removed to me.

Reply:
indeed that could be removed